backend-opal/ecosystem.config.js

```diff
@@ -2,18 +2,18 @@ module.exports = {
   apps: [
     {
       name: 'opal',
-      script: './main.js'
+      script: './start.js'
     }
   ],
   deploy: {
     production: {
       user: 'ubuntu',
       host: 'ec2-54-199-209-203.ap-northeast-1.compute.amazonaws.com',
-      key: '../../jsa-opal.pem',
+      key: '~/.ssh/jsa-opal.pem',
       ref: 'origin/master',
       repo: 'git@github.com:green-fox-academy/JSA-Opal-ShenZhen.git',
       path: '/home/ubuntu/opal',
-      'post-deploy': 'npm install && pm2 startOrRestart ecosystem.config.js'
+      'post-deploy': 'cd backend-opal && npm install && pm2 startOrRestart ecosystem.config.js'
```

Comment:
@hondahmd will this run the migrations after deploy?

Reply:
@adamgyulavari Added now. I've checked it on server and it works.